docs/SDD-AGENT-TEAMS.md

```
@@ -0,0 +1,612 @@
+conversación, pasan estas cosas:
+
+- El contexto se satura (Claude "olvida" lo que leyó al inicio)
+- No hay puntos de revisión humana — todo se ejecuta de golpe
+- Si la conversación se corta, se pierde todo el progreso
+- No hay documentación de las decisiones tomadas
+
+### La solución
+
+En vez de que un solo Claude haga todo:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)